components/Stands/StandCard.tsx

```diff
@@ -8,32 +8,42 @@ import dayjs from 'dayjs';
 import customParseFormat from 'dayjs/plugin/customParseFormat';
 import { timeIsBetween, timeIsAfter } from '../../utils/time';
 import { createFragmentContainer, graphql } from 'react-relay';
-import { StandCard_company } from '../../__generated__/StandCard_company.graphql';
+import { StandCard_stand } from '../../__generated__/StandCard_stand.graphql';
 
 dayjs.extend(customParseFormat);
 
 type Package = 'standard' | 'sp' | 'hsp';
+type standEvents = StandCard_stand["events"];
+type ArrayElement<ArrayType extends readonly unknown[]> = ArrayType[number];
+
 interface StandCardProps {
   type: Package;
   time: number;
-  company: StandCard_company;
+  stand: StandCard_stand | null;
```

Comment:
Why allow stand to be `null` in this component? `null` should be filtered out further up in the tree.

Reply:
This would also remove the need for all the null checks in this component.